plugins/woocommerce/src/Internal/BatchProcessing/BatchProcessingController.php

```diff
@@ -349,7 +352,15 @@ private function get_processor_instance( string $processor_class_name ): BatchPr
 	 * @return array List (of string) of the class names of the enqueued processors.
 	 */
 	public function get_enqueued_processors(): array {
-		return get_option( self::ENQUEUED_PROCESSORS_OPTION_NAME, array() );
+		$enqueued_processors = get_option( self::ENQUEUED_PROCESSORS_OPTION_NAME, array() );
+
+		if ( ! is_array( $enqueued_processors ) ) {
+			$this->logger->error( 'Could not fetch list of processors. Clearing up queue.', array( 'source' => 'batch-processing' ) );
+			delete_option( self::ENQUEUED_PROCESSORS_OPTION_NAME );
```

Comment:
Maybe deleting the option is a bit heavy handed? Leaving it alone won't break anything but maybe it's best to clean up?